test/plots/cars-mpg.js

```diff
@@ -0,0 +1,35 @@
+import * as Plot from "@observablehq/plot";
+import * as d3 from "d3";
+
+export default async function() {
+  const data = await d3.csv("data/cars.csv", d3.autoType);
+
+  return Plot.plot({
+    x: {inset: 20, ticks: 10},
+    y: {grid: true},
+    color: {
+      type: "categorical",
+      legend: true // as soon as it's available
+    },
+    marks: [
+      Plot.ruleY([0], {filter: false}),
```

Comment:
Is this filter doing anything?

```suggestion
      Plot.ruleY([0]),
```

Reply:
yes it's adding 0 in the domain, but no mark

Reply:
writing this I suddenly remember the zero: option :)

Reply:
Oh, fascinating. I wasn’t expecting this to work, but it does because Array.from(false) returns []. I think the zero option is more semantic so I switched to that.